devutils/src/devutils/invoke_utils.py

```diff
@@ -171,12 +172,15 @@ def run(ctx, *cmd, **options):
         args = " ".join(str(c) for c in cmd)
         return ctx.run(args, **options)
 
-    def poetry(ctx, *cmd):
+    def poetry(ctx, *cmd, verbose: bool = False):
         prefix = []
         if _use_conda():
             prefix.extend(["conda", "run", "--no-capture-output", "-n", CONDA_ENV_NAME])
         prefix.append("poetry")
 
+        if verbose:
+            prefix.append("--verbose")
```

Comment:
```suggestion
            prefix.append("-vv")
```
as one level of verbosity is not enough here. (doesn't spit too much)